src/Uno.UI/UI/Xaml/Internal/InputManager.Pointers.Managed.cs

```diff
@@ -226,6 +247,18 @@ private void OnPointerExited(Windows.UI.Core.PointerEventArgs args)
 
 		private void OnPointerPressed(Windows.UI.Core.PointerEventArgs args)
 		{
+			if (_inputManager._pointerRedirections?.TryGetValue(args.CurrentPoint.PointerId, out var recognizer) == true)
+			{
+#if HAS_UNO_WINUI
+				recognizer.ProcessDownEvent(new PointerPoint(args.CurrentPoint));
+#else
+				var pointer = args.CurrentPoint;
+				recognizer.ProcessDownEvent(new PointerPoint(
+					pointer.FrameId, pointer.Timestamp, pointer.PointerDevice, pointer.PointerId, pointer.RawPosition, pointer.Position, pointer.IsInContact, new PointerPointProperties(pointer.Properties)));
```

Comment:
We should add a cast operator or an extension method instead. (My pref would be for an `implicit` cast)

Reply:
I actually think `args.CurrentPointer` could be passed directly. I pushed a change and will see if it passes CI